### DISCUSSION TITLE
使用量算工具时渲染

### DISCUSSION THREAD

**Discussion by skyBlueRain** [zouyaoji/vue-cesium#637] [Q&A]
![1734336276589](https://github.com/user-attachments/assets/2085744d-ec65-4beb-8758-1c2f4a749a4c)
选择量算工具后，点击地图时就会弹出渲染中断？请问是什么欸？

**Comment by zouyaoji** [comment]
这是cesium新版本的api改变了，需要升级vue-cesium版本

**Comment by skyBlueRain** [comment]
其实cesium的版本是对的，但是里面在vue-cesium中很多地方调用的还是老版的wgs84ToWindowCoordinates方法，所以我很好奇这种情况是因为什么喃？有时间的话希望能解答一下




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zouyaoji/vue-cesium"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月17日(星期二) 下午4:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zouyaoji/vue-cesium] 使用量算工具时渲染 (Discussion #637)





 
这是cesium新版本的api改变了，需要升级vue-cesium版本

**Reply by zouyaoji** [reply]
你升级vue-cesium到最新版本应该就没这个问题了

**Comment by skyBlueRain** [comment]
好的，感谢回复


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zouyaoji/vue-cesium"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月17日(星期二) 下午4:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zouyaoji/vue-cesium] 使用量算工具时渲染 (Discussion #637)





 
你升级vue-cesium到最新版本应该就没这个问题了
